### DISCUSSION TITLE
what happens on delayed jobs when server crashes/restarts

### DISCUSSION THREAD

**Discussion by faysal515** [OptimalBits/bull#2282] [Q&A] [answered]
I have a use case of running a specific task after X seconds every time an API is hit. I'm planning to use Bull's `delayed` feature to wait before my task starts to execute. I was wondering what happens if I have for example 10 delayed jobs waiting to execute and in the interim my web server takes in a restart because of crash or new deployment?

**Comment by manast** [comment, marked as answer]
The job will be processed after the delayed time has passed, so when the server comes up again, if more than 10 seconds passed it will run the job.
